tests/boulder-integration.sh

```diff
@@ -14,7 +14,7 @@ export PATH="/usr/sbin:$PATH"  # /usr/sbin/nginx
 export GOPATH="${GOPATH:-/tmp/go}"
 export PATH="$GOPATH/bin:$PATH"
 
-if [ `uname`  == 'Darwin' ]; then
+if [ `uname` = "Darwin" ];then
```

Comment:
Sorry about that. Are we following a specific style guide for shell code? I looked up the [Google Style Guide for shell scripts](http://google.github.io/styleguide/shell.xml) and I can tell we aren't following that since it says to always use bash (though it does agree with you that it should be ```; then```). Just grepping for ```;then``` and ```; then``` on my system, I see plenty of both (in packages provided by Ubuntu).